src/main/java/org/gwtproject/dom/builder/client/DomStylesBuilder.java

```diff
@@ -111,19 +112,19 @@ static String toCamelCaseForm(String name) {
    * @param name the user specified style name
    * @return the camelCase name, or null if not set
    */
-  private static native String getCamelCaseName(JavaScriptObject map, String name) /*-{
-    return map[name] || null;
```

Comment:
Really curious what the thinking used to be here - just avoid `undefined` being returned? If that is a problem, seems like it could still be one (but I didn't think it was...)...

Reply:
Well, I'm just forwarding along @jnehlmeier original changes into our fork.  

The semantics may have changed because JsPropertyMap.get() has a cast in it, which likely coerces undefined -> null, or maybe it was superfluous to begin with.

Should be fairly straightforward to check, though I'm not sure it's worth the time?